esx_service/utils/auth.py

```diff
@@ -324,6 +326,25 @@ def add_volume_to_volumes_table(tenant_uuid, datastore, vol_name, vol_size_in_MB
 
     return None
 
+def remove_volume_from_volumes_table(tenant_uuid, datastore, vol_name):
+    """ Remove volume from volumes table. """
```

Comment:
Can we add a line about return code e.g. "Returns None on success or error string" to all functions?